File: 프로그래머스/lv1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.py
def solution(s):
    answer = True
    s = list(s)
    p_cnt = 0
    y_cnt = 0
    
    for i in s:
        if i == "p" or i == "P" :
            p_cnt += 1
            print(p_cnt)
        elif i == "y" or i == "Y":
            y_cnt += 1
        
    if p_cnt == y_cnt :
        return True
    else : 
        return False

File: 프로그래머스/lv1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.java
class Solution {
    boolean solution(String s) {
        int yCnt = 0;
        int pCnt = 0;

        for (String n : s.split("")) {
            if (n.equalsIgnoreCase("y")) {
                yCnt++;
            } else if (n.equalsIgnoreCase("p")) {
                pCnt++;
            }
        }

        return yCnt == pCnt;
    }
}